Artifacts/PgSql/Data/Security/1060-RemoveRedundantBootstrapPermissions.sql

```diff
@@ -0,0 +1,49 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DO language plpgsql $$
+DECLARE
+    educationOrganizationsResourceClaimId INT;
+	claimSet_Id INT;
+	claim_Name VARCHAR(250) := 'http://ed-fi.org/ods/identity/claims/domains/educationOrganizations';
+	claimSet_Name VARCHAR(250) := 'Bootstrap Descriptors and EdOrgs';	
+BEGIN
+   
+   IF NOT EXISTS (SELECT 1 FROM dbo.ResourceClaims WHERE ClaimName = claim_Name)
+   THEN
+        RAISE NOTICE '% does not exist', claim_Name;
+   END IF;
+   
+   	IF NOT EXISTS (SELECT 1 FROM dbo.ClaimSets WHERE ClaimSetName = claimSet_Name)
+   	THEN
+       	RAISE NOTICE '% does not exist', claimSet_Name;
+   	END IF;
+	
+   IF EXISTS (SELECT 1 FROM dbo.ResourceClaims WHERE ClaimName = claim_Name)
+   	THEN
+        SELECT ResourceClaimId INTO educationOrganizationsResourceClaimId
+        FROM dbo.ResourceClaims
+        WHERE ClaimName = claim_Name;
+		
+   		IF EXISTS (SELECT 1 FROM dbo.ClaimSets WHERE ClaimSetName = claimSet_Name)
+    	 THEN
+        	SELECT claimSetId INTO claimSet_Id
+        	FROM dbo.ClaimSets
+        	WHERE ClaimSetName = claimSet_Name;
+   		 END IF;
+		
+		DELETE FROM dbo.ClaimSetResourceClaims 
+		WHERE ClaimSetResourceClaimId IN  
+		(SELECT ClaimSetResourceClaimId
+		FROM dbo.ClaimSetResourceClaims 
+		WHERE ClaimSet_ClaimSetId = claimSet_Id
+		AND ResourceClaim_ResourceClaimId IN 
```

Comment:
Add criteria for _Action_ActionId_ and _AuthorizationStrategyOverride_AuthorizationStrategyId_ here as well, as described above for SQL Server.

Reply:
I added here also .Thanks